src/setops.jl

```diff
@@ -74,11 +74,11 @@ function _reconstruct(method::TaylorModelReconstructor, P₀::LazySet, U₀::Laz
     zeroI = interval(zero(N), zero(N))
     Δtn = zeroI
     for i in 1:n
-        rem = remainder(S[i])
-        W = TaylorModelN(X_Δt[i], rem, zeroBox(n + m), symBox(n + m))
+        W = TaylorModelN(X_Δt[i], zeroI, zeroBox(n + m), symBox(n + m))
```

Comment:
IIUC the second argument to the `TaylorModelN` constructor should be the zero interval, because the remainder it's already taken into accout in the evaluation of `X_Δt[i]`. (otherwise, it's taken twice).

cc @lbenet

Reply:
Great that you found this :confetti_ball: